### PR TITLE
SSB pre release

### PR DESCRIPTION
**Pull request by alanshaw**
Basically just a test to see how we can create a scuttlebot for the app and how we might programmatically create an invite and accept it and receive messages on another scuttlebot.

Next steps are probably to have scuttlebot started as a process by systemd/upstart and use `ssb-client` to interact with it instead.

### CODE REVIEW

**Comment by richsilv** [comment]
:eyeglasses:

File: ssb-pre-release/test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  createSbotServer(APP_NAME, { temp: true, keys: ssbKeys.generate(), allowPrivate: true }, (err, primary) => {
+    t.ifError(err, 'server started')
+
+    createSbotServer(APP_NAME, { temp: true, keys: ssbKeys.generate() }, (err, secondary) => {
+      t.ifError(err, 'follower started')
+
+      primary.invite.create(1, (err, invite) => {
+        t.ifError(err, 'invite created')
+
+        secondary.invite.accept(invite, (err, res) => {
```
Comment (richsilv) [inline ssb-pre-release/test.js:48]: This is getting a bit callback-triangley, but I'll leave it to you to decide if that's cleaner than async/promisifying.

**Comment by richsilv** [comment]
I get an error of this form when running the tests, which all pass:

```sh
Error replicating with @QPehOnTiNI6oFFDdvaTUIxNJL3hujIhym70z/ZkodfA=.ed25519:
Error: no source:createHistoryStream
```

Assuming this is nothing to worry about, crack on and merge.

**Review by olizilla — APPROVED** [review]
FUTURE TECH

**Pull request merged**